HoloPipelines/pipelines/components/compNifti2numpy.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import nibabel as nib
+
+
+def resample(dataPath, new_spacing=[1, 1, 1]):
+    image = dataPath
+    originalShape = image.shape[:3]
+    header = image.header
```

Comment:
local variable 'header' is assigned to but never used

Reply:
`header = image.header` deleted